src/Pact/Native/Ops.hs

```diff
@@ -182,10 +182,18 @@ logDef = defRNative "log" log' coerceBinNum ["(log 2 256)"] "Log of Y base X."
   log' fi as@[TLiteral base _,TLiteral v _] = do
     unlessExecutionFlagSet FlagDisablePact43 $
       when (not (litGt0 base) || not (legalLogArg v)) $ evalError' fi "Illegal base or argument in log"
+#if defined(ghcjs_HOST_OS)
     binop (\a b -> liftDecF fi logBase a b)
           (\a b -> liftIntF fi logBase a b)
           fi
           as
+#else
+    decimalLogBase <-
+      ifExecutionFlagSet' FlagDisableNewPow (liftDecF fi logBase) (liftDecF fi trans_log)
```

Comment:
Why `liftDecF fi trans_log` and not `unopd trans_log` similar to the other cases below?